repository/repository.go

```diff
@@ -156,17 +156,26 @@ func RetrieveChangeLogContents(ctx context.Context, client *github.Client, repo,
 			body := prs[0].GetBody()
 
 			var releaseNote string
+			inNote := false
 			if strings.Contains(body, releaseNoteSection) && !strings.Contains(body, emptyReleaseNote) {
 				lines := strings.Split(body, "\n")
-				for i, line := range lines {
+				for _, line := range lines {
 					if strings.Contains(line, releaseNoteSection) {
-						if lines[i+1] == "```" || lines[i+1] == "" {
-							continue
-						}
-						releaseNote += lines[i+1]
+						inNote = true
+						continue
+					} else if strings.Contains(line, "```") {
```

Comment:
I don't believe the `else` is necessary in this case.

Reply:
How would you drop out of the release note then?